src/corefx/System.Globalization.Native/configure.cmake

```diff
@@ -10,6 +10,9 @@ CHECK_CXX_SOURCE_COMPILES("
 
 if(NOT CLR_CMAKE_PLATFORM_DARWIN)
     set(CMAKE_REQUIRED_LIBRARIES ${ICUUC} ${ICUI18N})
+    if(CLR_CMAKE_PLATFORM_UNIX_ARM)
+        set(CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES} icudata m stdc++ dl)
```

Comment:
@seanshpark, can you reproduce this issue on other arm devices with gcc-5.3? Or could it be the local misconfiguration (some bad exports etc.) requiring this? Would be nice if you could reconfirm.